taichi/transforms/constant_fold.cpp

```diff
@@ -208,6 +208,7 @@ void constant_fold(IRNode *root) {
     TI_TRACE("config.debug enabled, ignoring constant fold");
     return;
   }
+  return;  // XXX: TRY
```

Comment:
remove?

Reply:
Sorry, I had a hard time to make CI happy.

Reply:
You mean without this `return`, we get the CI error? Is it the one in https://github.com/taichi-dev/taichi/pull/1023#issuecomment-631294228 ?